pkg/types/core/standard_capabilities_services.go

```diff
@@ -0,0 +1,16 @@
+package core
+
+// StandardCapabilitiesServices contains all the services required for capability initialization.
+// We use a struct to evolve the interface without requiring updates to all implementors.
+type StandardCapabilitiesServices struct {
```

Comment:
How about a more general name that also avoids collisions with the `services` packages?
```suggestion
// StandardCapabilitiesDependencies contains all the dependencies injected for capability initialization.
// We use a struct to evolve the interface without requiring updates to all implementors.
type StandardCapabilitiesDependencies struct {
```

Reply:
(or `InitialiseRequest` to match the grpc layer underneath 🤷 )

Reply:
ec1247e49492f16c8a062c5ec2cdd918f878c17e